camel-frappe-api/src/main/java/com/ozonehis/camel/frappe/sdk/internal/security/BasicAuthentication.java

```diff
@@ -0,0 +1,80 @@
+package com.ozonehis.camel.frappe.sdk.internal.security;
+
+import com.ozonehis.camel.frappe.sdk.api.security.FrappeAuthentication;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.List;
+
+@Slf4j
+public class BasicAuthentication implements FrappeAuthentication {
```

Comment:
"Basic Authentication" usually means [this scheme](https://datatracker.ietf.org/doc/html/rfc7617). I'd probably call this "DefaultAuthentication" or "DefaultFrappeAuthentication".

Reply:
`DefaultFrappeAuthentication` makes sense to me.